omod/src/main/webapp/fragments/header.gsp

```diff
@@ -114,7 +116,7 @@
         <li class="change-location">
```

Comment:
Why didn't you end up with simply adding:
<input id="sessionLocationUuid" type="hidden" value="${ sessionContext.sessionLocation.uuid }">
?